Formula/freeimage.rb

```diff
@@ -25,6 +25,13 @@ class Freeimage < Formula
   end
 
   def install
+    # Temporary workaround for ARM. Upstream tracking issue:
+    # https://sourceforge.net/p/freeimage/bugs/325/
+    # https://sourceforge.net/p/freeimage/discussion/36111/thread/cc4cd71c6e/
+    if Hardware::CPU.arm?
+      ENV.append "CFLAGS", "-DPNG_ARM_NEON_OPT=0"
```

Comment:
This is actually changing the environment more than you expect due to the way the makefile works. You probably want something like this instead:
```suggestion
    ENV["CFLAGS"] = "-O3 -fPIC -fexceptions -fvisibility=hidden -DPNG_ARM_NEON_OPT=0" \
      if build.stable?
```